header.php

```diff
@@ -50,16 +50,9 @@
 							// Edit post link.
 								edit_post_link(
 									sprintf(
-										wp_kses(
-											/* translators: %s: Name of current post. Only visible to screen readers. */
-											__( 'Edit <span class="screen-reader-text">%s</span>', 'twentynineteen' ),
-											array(
-												'span' => array(
-													'class' => array(),
-												),
-											)
-										),
-										get_the_title()
+										/* translators: %s: Name of current post. Only visible to screen readers. */
+										esc_html__( 'Edit %s', 'twentynineteen' ),
+										'<span class="screen-reader-text">' . get_the_title() . '</span>'
```

Comment:
This change needs to be reverted to support proper translations in all languages. See https://core.trac.wordpress.org/ticket/31979 and  https://github.com/WordPress/twentysixteen/pull/383 for background.